docs/admin_api/user_admin_api.md

```diff
@@ -773,6 +773,43 @@ Note: The token will expire if the *admin* user calls `/logout/all` from any
 of their devices, but the token will *not* expire if the target user does the
 same.
 
+## Allow replacing master cross-signing key without User-Interactive Auth
+
+This endpoint is not intended for server administrator usage;
+we describe it here for completeness.
```

Comment:
Wasn't sure if I should omit this. But I thought it might be best to err on the side of transparency.

Reply:
I'm happy to keep this documented, it's clear enough that as an admin you shouldn't use that